examples/ex22p.cpp

```diff
@@ -0,0 +1,314 @@
+//                       MFEM Example 5 - Parallel Version
```

Comment:
Can you add also the serial version?

Reply:
There is no serial version yet, but I will try if the dependencies are in check for one.